--- /etc/pam.d/gdm-password	2022-01-12 06:15:56.000000000 -0800
+++ gdm-password-keyring	2022-02-02 11:45:55.373050369 -0800
@@ -7,7 +7,7 @@
 account     include       password-auth
 
 password    substack       password-auth
--password   optional       pam_gnome_keyring.so use_authtok
+password   optional       pam_gnome_keyring.so use_authtok
 
 session     required      pam_selinux.so close
 session     required      pam_loginuid.so
