--- panel.js.orig	2013-07-26 15:39:35.032923977 -0700
+++ panel.js	2013-07-26 15:40:45.108247990 -0700
@@ -918,7 +918,7 @@
     'activities': ActivitiesButton,
     'appMenu': AppMenuButton,
     'dateMenu': imports.ui.dateMenu.DateMenuButton,
-    'a11y': imports.ui.status.accessibility.ATIndicator,
+//    'a11y': imports.ui.status.accessibility.ATIndicator,
     'a11yGreeter': imports.ui.status.accessibility.ATGreeterIndicator,
     'volume': imports.ui.status.volume.Indicator,
     'battery': imports.ui.status.power.Indicator,
@@ -928,9 +928,9 @@
     'userMenu': imports.ui.userMenu.UserMenuButton
 };
 
-if (Config.HAVE_BLUETOOTH)
-    PANEL_ITEM_IMPLEMENTATIONS['bluetooth'] =
-        imports.ui.status.bluetooth.Indicator;
+//if (Config.HAVE_BLUETOOTH)
+//    PANEL_ITEM_IMPLEMENTATIONS['bluetooth'] =
+//        imports.ui.status.bluetooth.Indicator;
 
 try {
     PANEL_ITEM_IMPLEMENTATIONS['network'] =
